crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2217,6 +2313,168 @@ swwwwwwwwwwwwwwww
 swwwwwwwwwwwwww
 ENDMAP
 
+NAME:   nicolae_shoals_bring_sunscreen
+TAGS:   transparent water_ok no_tide
+DEPTH:  Shoals:2-
+KMONS:  D = sun demon
+KMONS:  M = sun moth
+SUBST:  { = {[(, x = cxx..., _ = _.
+NSUBST: S = M / 2=MDD / 2=D / 2=D. / .
+CLEAR: _
+MAP
+      _....._
+    _........._
+    ...S...S...
+   _..........._
+ _..S...xxx...S.._
+ ......xxcxx......
+_.......xxx......._
+..S..x.......x..S..
+....xxx.....xxx....
+....xcx..{..xcx....
+....xxx.....xxx....
+..S..x.......x..S..
+_.......xxx......._
+ ......xxcxx......
+ _..S...xxx...S.._
+   _..........._
+    ...S...S...
+    _........._
+      _....._
+ENDMAP
+
+NAME:   nicolae_shoals_offshore_bank_account
+TAGS:   water_ok no_pool_fixup transparent
+DEPTH:  Shoals
+ORIENT: west
+KITEM:  pr = $ w:100 / any jewellery
+KITEM:  q = $ w:50 / $ good_item w:30 / any jewellery good_item / \
+            any weapon good_item w:2 / any armour good_item w:2
+KMONS:  p = merfolk siren / nothing w:60
+KMONS:  q = merfolk impaler / merfolk javelineer / merfolk aquamancer / \
+            nothing w:30
+KMONS:  r = water nymph
+KFEAT:  ~ = open_sea
+KPROP:  =`pqr = no_tele_into
+KPROP:  =`pqr = no_tide
+KMASK:  =`pqr = opaque
+KMASK:  =`pqr = no_trap_gen
+KMASK:  =`pqr = no_monster_gen
+SUBST:  M = w_, N = wwW.
+CLEAR:  _
+MAP
+      MwwM
+    MwwwwwwM
+  MwwwwwwwwwwM
+~wwwwwwwwwwwwwwM
+~wwwwwwwwwwwwwwwwM
+~wwwwwcwwwcwwwwwwww
+~wwwwcccccccwwwwwNN
+~wwwcccqqqcccwwwN...
+~wwwwc`pppqcwwwN.....
+~wwww=`prpqcwwwN....@
+~wwwwc`pppqcwwwN.....
+~wwwcccqqqcccwwwN...
+~wwwwcccccccwwwwwNN
+~wwwwwcwwwcwwwwwwww
+~wwwwwwwwwwwwwwwwM
+~wwwwwwwwwwwwwwM
+  MwwwwwwwwwwM
+    MwwwwwwM
+      MwwM
+ENDMAP
+
+NAME:   nicolae_shoals_deep_elves_summer_vacation
+TAGS:   water_ok transparent patrolling no_pool_fixup
+DEPTH:  Shoals
+ORIENT: float
+# Someone left their gear in the changing room.
+KITEM:  P = leather armour / ring mail / scale mail / chain mail, \
+             long sword / short sword / rapier / dagger, \
+             buckler / nothing w:20
+# Some people like having a little something to read or drink on the beach.
+KITEM:  12 = nothing w:30 / any book / any scroll / any potion
+# Deep elves have to bring the high-strength sunscreen. At least one potion
+# of resistance is guaranteed in the changing room.
+KITEM:  s = potion of resistance ident:all
+# 1/3 chance of rudely walking in on a deep elf changing.
+KMONS:  p = deep elf knight ; nothing / nothing w:20
```

Comment:
The philosophy of "shouldna been standin there" applies to everyone, not just the player. I've changed the layout so that the changing elf is standing on the same tile as the gear, which hopefully will inspire them to put it back on if they hear something going down. Or not.